src/main/resources/META-INF/plugin.xml

```diff
@@ -1,14 +1,14 @@
 <!-- Plugin Configuration File. Read more: https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html -->
-<idea-plugin>
+<idea-plugin require-restart="true">
     <!-- Unique identifier of the plugin. It should be FQN. It cannot be changed between the plugin versions. -->
-    <id>com.software.codetime</id>
+    <id>com.softwareco.intellij.plugin</id>
 
     <!-- Public plugin name should be written in Title Case.
          Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-name -->
-    <name>CodeTime</name>
+    <name>Code Time</name>
 
     <!-- A displayed Vendor name or Organization ID displayed on the Plugins Page. -->
-    <vendor url="https://www.software.com">Software.com</vendor>
+    <vendor url="https://www.software.com">Software</vendor>
```

Comment:
lets link directly to the CodeTime page of our website so its more specific to this product

Reply:
cool, will do